docs/design/datacontracts/Exception.md

```diff
@@ -1,28 +1,56 @@
-# Contract Thread
+# Contract Exception
 
 This contract is for getting information about exceptions in the process.
 
 ## APIs of contract
 
+```csharp
+record struct ManagedExceptionData(
```

Comment:
Should we call this just `ExceptionData`? 

I think it would be best to avoid `Managed` prefix in the contract definitions.

Reply:
If there is something explicitly unmanaged (like `EXCEPTION_INFORMATION` - that is unmanaged equivalent of managed exception), I would use a special prefix for that if necessary.